src/connectionmonitor/SignalingAndMetricsConnectionMonitor.ts

```diff
@@ -116,9 +116,23 @@ export default class SignalingAndMetricsConnectionMonitor
     if (packetsReceived === 0 || fractionPacketsLostInbound > 0) {
       this.connectionHealthData.setLastPacketLossInboundTimestampMs(Date.now());
     }
+    const potentialAudioPacketsSent = metricReport.audioPacketsSent;
```

Comment:
nit: why the 'potential' prefix?

Reply:
Was just following the norm set by the code written above. But since it's a bit confusing, I'll just remove the declaration and just check whether the `audioPacketsSent` metric is present or not and use it